frameit/lib/frameit/frame_converter.rb

```diff
@@ -64,6 +64,27 @@ def convert_frames
 
         UI.important "Converting PSD file '#{psd}'"
         image = MiniMagick::Image.open(psd)
+
+        if psd =~ /iPhone-SE/
+          UI.success "Removing white background 🚫 ⬜️"
+
+          # The iPhone-SE screenshots from April 2016 have
+          # 3 layers, a background, a product, and the 'put your image here' layer
+          # imagemagick seems to add an additional layer with no label which this the
+          # composite of all three.  We want to remove the background and composite layer
+          good_layers = image.layers.reject do |layer|
+            label = layer.details['Properties']['label']
+            label =~ /White B/i || label.to_s.length == 0
```

Comment:
might want to reverse the order of these comparisons in case the label is ever `nil`